src/main/webapp/app/exercises/file-upload/assess/file-upload-assessment-dashboard.component.html

```diff
@@ -3,6 +3,12 @@ <h4 jhiTranslate="artemisApp.fileUploadAssessment.dashboard.heading" [translateV
     <jhi-assessment-filters [submissions]="submissions" (filterChange)="updateFilteredSubmissions($event)"></jhi-assessment-filters>
     <ng-container *ngIf="!busy"></ng-container>
     <jhi-assessment-warning [exercise]="exercise" [submissions]="filteredSubmissions"></jhi-assessment-warning>
+    <h4>
+        <small [ngClass]="assessedSubmissions === submissions.length && submissions.length > 0 ? 'text-success' : 'text-warning'">
+            <strong>Progress: {{ (assessedSubmissions / submissions.length) * 100 | number: '0.0-0' }} % </strong>({{ assessedSubmissions }} out of {{ submissions.length }} upload
+            exercises assessed)
+        </small>
+    </h4>
```

Comment:
I don't think we should hardcode this since it contains natural language. Please provide translation keys for it so that it aligns with the current language settings in Artemis.

Reply:
Thank you @willergeorg and @Dominik-Weinzierl. 
I have implemented the suggestions. Could you take a second look? 
Thank you in advance!